about.html

```diff
@@ -75,6 +78,14 @@ <h2 id="getinvolved">Get Involved</h2>
 						<a class="button button-subtle" href="action.html#list">Join newsletter</a>
 					</p>
 				</div>
+				<script>
+					let elements = document.getElementsByClassName("header-copy");
+					for (let i = 0; i < elements.length; i++) {
+						let element = elements[i];
+						let parentId = element.parentElement.id;
+						addCopyButton(element, location.protocol + '//' + location.host + location.pathname + "#" + parentId)
+					}
+				</script>
```

Comment:
If we're adding this <script> to almost every page would it make sense to abstract it to a file instead of create it 4+ times?